tests/integration-tests/tests/dcv/test_dcv.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import boto3
+import pytest
+
+from assertpy import assert_that
+from remote_command_executor import RemoteCommandExecutor
+
+SERVER_URL = "https://localhost"
+
+
+def _check_auth_ko(remote_command_executor, dcv_authenticator_port, params, expected_message):
+    assert_that(
+        remote_command_executor.run_remote_command(
+            f"curl -s -k -X GET -G {SERVER_URL}:{dcv_authenticator_port} {params}"
+        ).stdout
+    ).contains(expected_message)
+
+
+def _check_shared_dir(remote_command_executor, shared_dir):
+    assert_that(
+        int(remote_command_executor.run_remote_command(f"cat /var/log/dcv/server.log | grep -c {shared_dir}").stdout)
+    ).is_greater_than(0)
+
+
+def _check_auth_ok(remote_command_executor, external_authenticator_port, session_id, session_token):
+    assert_that(
+        remote_command_executor.run_remote_command(
+            f"curl -s -k {SERVER_URL}:{external_authenticator_port} "
+            f"-d sessionId={session_id} -d authenticationToken={session_token} -d clientAddr=someIp"
+        ).stdout
+    ).is_equal_to('<auth result="yes"><username>centos</username></auth>')
+
+
+def _check_security_group(region, cluster, port, expected_cidr):
+    security_group_id = cluster.cfn_resources.get("MasterSecurityGroup")
+    response = boto3.client("ec2", region_name=region).describe_security_groups(GroupIds=[security_group_id])
+
+    ips = response["SecurityGroups"][0]["IpPermissions"]
+    target = next(filter(lambda x: x.get("FromPort", -1) == port, ips), {})
+    assert_that(target["IpRanges"][0]["CidrIp"]).is_equal_to(expected_cidr)
+
+
+@pytest.mark.parametrize(
+    "dcv_port, access_from, shared_dir", [(8443, "0.0.0.0/0", "/shared"), (5678, "192.168.1.1/32", "/myshared")]
+)
+@pytest.mark.regions(["eu-west-1"])
+@pytest.mark.oss(["centos7"])
+@pytest.mark.schedulers(["sge"])
```

Comment:
Why just SGE?

Reply:
Fixed by adding all the supported OSes and also awsbatch